engine/src/main/java/org/terasology/reflection/copy/strategy/ListCopyStrategy.java

```diff
@@ -33,11 +34,7 @@ public ListCopyStrategy(CopyStrategy<T> contentStrategy) {
     @Override
     public List<T> copy(List<T> value) {
         if (value != null) {
-            List<T> result = Lists.newArrayListWithCapacity(value.size());
-            for (T item : value) {
-                result.add(contentStrategy.copy(item));
-            }
-            return result;
+            return value.stream().map(contentStrategy::copy).collect(Collectors.toCollection(ArrayList::new));
```

Comment:
The method `Collectors.toList()` should do the trick - is there a reason to explicitly specify `ArrayList`?